tuist/tuist
# Add `tuist inspect implicit-dependencies` to detect implicit dependencies

Part of https://github.com/tuist/tuist/issues/6492

### Short description 📝

> Describe here the purpose of your PR.

The purpose of this PR is to find way to get all the imports of the module to check it later on implicitness. How @kwridan noticed in [comment](https://github.com/tuist/tuist/issues/6492#issuecomment-2213413457), the best way to do it is static analysis and pattern matching. Also I added ImportFinderCommand in purpose of testing ImportFinderService. I tested it on Tuist project and it works, but still a lot of things unclear, for example, targets with the same names and other corner cases.

### How to test the changes locally 🧐

Launch import_finder command with --path and --target-name options. For example, 
 tuist lint implicit-imports --path tuist/fixtures/ios_app_with_frameworks

> Include a set of steps for the reviewer to test the changes locally (see [the documentation](https://docs.tuist.io/contributors/get-started) for reference).

### Contributor checklist ✅


### Reviewer checklist ✅
